package.json

```diff
@@ -4,6 +4,7 @@
   "description": "## Description Benchmark goals are projects that are designed to be done solo.",
   "main": "index.js",
   "scripts": {
+    "start": "node ./scripts/guess.js",
```

Comment:
give alias instead of start to run this script. Maybe "Guess" which you could run by calling ~npm run guess . This will allow you to give a different script alias to each of the benchmark goals.